ui/app/src/Utils/Filter.elm

```diff
@@ -231,6 +232,31 @@ stringifyMatcher { key, op, value } =
         ++ "\""
 
 
+convertFilterMatchers : List Matcher -> List Data.Matcher.Matcher
+convertFilterMatchers matchers_ =
+    case matchers_ of
+        [] ->
+            []
+
+        list ->
+            list
+                |> List.map convertFilterMatcher
```

Comment:
I don't think we need the `convertFilterMatchers` function, because it can be replaced with `List.map convertFilterMatcher matchers`. 

There is no need to check for the empty list, because `List.map convertFilterMatcher [] == []`

Reply:
thanks for your review. I will remove it.